x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/engine_table.tsx

```diff
@@ -64,7 +68,9 @@ export const EngineTable: ReactFC<IEngineTableProps> = ({
     },
     {
       field: 'created_at',
-      name: 'Created At',
+      name: i18n.translate('xpack.appSearch.enginesOverview.table.column.createdAt', {
+        defaultMessage: 'Created At',
+      }),
       dataType: 'string',
       render: dateString => {
```

Comment:
NOTE: I actually think I'm missing something here and need to use react-intl's `FormattedDate` helper ☹️ I'll come back to this on Monday.

Reply:
Edit: Yup, https://formatjs.io/docs/react-intl/components/#formatteddate and https://formatjs.io/docs/react-intl/components/#formattednumber works a charm. Does all the hard work for me!